### PR TITLE
Lobby screen... changes? improvements? I'm bad at writing titles.

### PR DESCRIPTION
**Pull request by BenLubar**
## About the PR

- Defines the directions of the walls around the dream bee in the map so the walls don't look bad until world generation completes.
- Removes the title screen turf. Why was that a turf?
- Makes the loading progress and server list show up on image loading screens.
- Converts all map and game mode loading screens to images that work like the heisenbee reward one.

## Why's this needed?

Heisenbee loading screen was intended to be a reward for keeping the RD's little buddy alive, but it meant you couldn't see the loading progress. Admin gimmicks on the map loading screen are now easier to set up as the image (HTML) loading screen is only shown to `/mob/new_player`.

### CODE REVIEW

File: code/WorkInProgress/zamujasa.dm [hunk 1]
```diff
@@ -944,53 +944,46 @@ Read the rules, don't grief, and have fun!</div>"}
...
-		else
-			// oops
-			src.x = 7
-			src.y = 2
-			src.z = 1
-			src.layer = 1
+			// let's just hard-code coordinates
+		src.x = 295
+		src.y = 19
+		src.z = Z_LEVEL_ADVENTURE
```
[the remark this answers is not in the document]
Comment (pali6) [reply code/WorkInProgress/zamujasa.dm:954]: Can I suggest this being a landmark? Then you could just do `src.set_loc(landmarks[LANDMARK_LOBBY_STATS])` or something.

File: code/WorkInProgress/zamujasa.dm [hunk 2]
```diff
@@ -944,53 +944,46 @@ Read the rules, don't grief, and have fun!</div>"}
...
-				src.layer = lobby_titlecard.layer + 1
-			else
-				// oops
-				src.x = 7
-				src.y = 1
-				src.z = 1
-				src.layer = 1
+			// what could possibly go wrong
+			src.x = 295
+			src.y = 20
```
Comment (pali6) [inline code/WorkInProgress/zamujasa.dm:984]: Same here.

File: code/WorkInProgress/zamujasa.dm [hunk 3]
```diff
@@ -1004,9 +997,42 @@ Read the rules, don't grief, and have fun!</div>"}
...
+				src.set_text("<span class='c ol vga vt'>Round begins<br><span style='color: #aaaaaa; font-size: 3em;'>soon</span></span>")
+
+	encourage
+		maptext_area = "leftside"
+
+		New()
+			..()
+			// apart from "everything", that is
+			src.x = 278
+			src.y = 19
```
Comment (pali6) [inline code/WorkInProgress/zamujasa.dm:1011]: And here? Idk.

**Comment by BenLubar** [comment]
what the-- how did I cause *that* runtime?

File: code/modules/admin/admin.dm [hunk 4]
```diff
@@ -827,15 +827,12 @@ var/global/noir = 0
...
+						lobby_titlecard = new /datum/titlecard/disaster()
 					else
-						lobby_titlecard.icon_state = "title_main"
-					#ifdef MAP_OVERRIDE_OSHAN
-						lobby_titlecard.icon_state = "title_oshan"
-					#endif
-					#ifdef MAP_OVERRIDE_MANTA
-						lobby_titlecard.icon_state = "title_manta"
-					#endif
+						lobby_titlecard = new /datum/titlecard()
```
[the remark this answers is not in the document]
Comment (ZeWaka) [reply code/modules/admin/admin.dm:836]: ?

[the remark this answers is not in the document]
Comment (pali6) [reply code/modules/admin/admin.dm:836]: It won't get rid of Heisenbee as it has `is_game_mode` set to `FALSE`.

**Review by pali6 — APPROVED** [review]

**Pull request merged**